src/functions/routers/action_router.ts

```diff
@@ -75,7 +76,14 @@ export class ActionsRouter<
    */
   addHandler(
```

Comment:
Is there a reason we're leaving this in when we type `routes` on line 56? Still looks like

```ts
private routes: Array<
    [BlockActionConstraint, BlockActionHandler<typeof this.func.definition>]
  >;
```

Reply:
@shapirone - Yes, TS seems to have no complains with using `this`, I guess because in the other cases we were referencing `this` in the return types of public methods. I think we can leave this alone.